browser/ui/toolbar/brave_app_menu_model.cc

```diff
@@ -247,12 +247,6 @@ void BraveAppMenuModel::InsertBraveMenuItems() {
   }
 #endif
 
-  // Insert adblock menu at last. Assumed this is always enabled.
-  DCHECK(IsCommandIdEnabled(IDC_SHOW_BRAVE_ADBLOCK));
-  InsertItemWithStringIdAt(GetIndexOfBraveAdBlockItem(),
```

Comment:
`GetIndexOfBraveAdBlockItem()` method also should be deleted?

Reply:
looks like it is used to figure out the index for ipfs menu item

```
int index = IsCommandIdEnabled(IDC_SHOW_BRAVE_SYNC)
                    ? GetIndexOfBraveSyncItem() + 1
                    : GetIndexOfBraveAdBlockItem();
    InsertSubMenuWithStringIdAt(index, IDC_APP_MENU_IPFS, IDS_APP_MENU_IPFS,
                                &ipfs_submenu_model_);
```

Reply:
I think we can just rename it to `GetLastIndexOfSecondSection()` because adblock item is positioned at last in second section.

Reply:
done